README.md

```diff
@@ -1,2 +1,292 @@
-# myproject1
-hello techwise project1
+
+<a name="readme-top"></a>
+
+<!--
+!!! IMPORTANT !!!
+This README is an example of how you could professionally present your codebase. 
+Writing documentation is a crucial part of your work as a professional software developer and cannot be ignored. 
+
+You should modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+
+For more information on the importance of a professional README for your repositories: https://github.com/Techwiseinc/curriculum-transversal-skills/blob/main/documentation/articles/readme_best_practices.md
+-->
```

Comment:
@Hammadzulfiqarx Good job adding a professional Readme but it would be great if you remove all the commented part form your readme as it will not create a good impact on your viewers. 🤙